clusterbuster

```diff
@@ -436,6 +443,19 @@ $(print_workloads_supporting_reporting '                        - ')
                         Allow use of direct I/O for virtiofsd (default $virtiofsd_direct).
        --virtiofsd-threadpoolsize=n
                         Use the specified thread pool size for virtiofsd (default 1).
+       --vmthreads=<value>
+                        Specify the number of threads on each core (default $vm_threads).
```

Comment:
Please put this in a separate section.  Please also rename "Virtualization Options" to "Kata Virtualization Tuning".

Reply:
Just did as suggested.